src/libraries/System.Net.Primitives/ref/System.Net.Primitives.cs

```diff
@@ -516,9 +516,11 @@ public enum SslProtocols
         Ssl2 = 12,
         [System.ObsoleteAttribute("SslProtocols.Ssl3 has been deprecated and is not supported.")]
         Ssl3 = 48,
+        [System.ObsoleteAttribute("TLS versions 1.0 and 1.1 are obsolete and not considered secure by the industry. Consider using newer TLS protocol version.", DiagnosticId = "SYSLIB0039", UrlFormat = "https://aka.ms/dotnet-warnings/{0}")]
```

Comment:
I'm wondering if we should use same wording as above for consistency. As minimum, they had the protocol names separate. I'm not sure if this comes from the new process. 
And when time comes to obsolete Tls 1.2 are we going to add new entry or are we going to add reference to SYSLIB0039 as well? 
Also updating Ssl2 & Ssl3 would be yet another breaking change but would it be worth to make it consistent? 

any thoughts on this @jeffhandley @bartonjs ?

Reply:
I will be happy to change the wording to whatever you find most appropriate :)

Reply:
@terrajobst For wording, and saying what our policy is on adding things to existing obsoletion codes.  (My guess is that we don't add things to an existing code)

Reply:
> My guess is that we don't add things to an existing code

I agree. Unless we also mark it as obsolete during .NET 7, we will use a new diagnostic ID for TLS 1.2. We can group things together when deprecating them all during the same release, but over multiple releases I expect we'll use new codes.

Here's what I recommend for wording. We can use the same message for both versions similar to how we've referenced multiple API members in messages.

```suggestion
        [System.ObsoleteAttribute("TLS versions 1.0 and 1.1 have known vulnerabilities and are not supported. Use a newer TLS version instead.", DiagnosticId = "SYSLIB0039", UrlFormat = "https://aka.ms/dotnet-warnings/{0}")]
```

Reply:
AFAIK our current guidance is to use `SslProtocols.None` e.g. defer the version selection to OS. I'm wondering if the message should reflect that. 
And "are not supported" seems  strong. While weak from security prospective I think we would still support customers who has issue with that particular TLS version. (but I'm ok if we pretend we don't)

Reply:
I wasn't sure about "are not supported" either. If someone were to claim the protocol vulnerability needed to somehow be patched by us, we'd of course reject such an issue. But otherwise, they are expected to "work" and be supported. Perhaps dialing it back to "... are not recommended."?

And yes, the second sentence should give whatever our best guidance is.

Reply:
So I guess "TLS versions 1.0 and 1.1 have known vulnerabilities and are not recommended. Use a newer TLS version instead, or use SslProtocols.None to defer to OS defaults." would be fine?

Reply:
That sounds good to me, @rzikm. Thanks!